disnake/http.py

```diff
@@ -1015,7 +1015,7 @@ def start_thread_with_message(
         message_id: Snowflake,
         *,
         name: str,
-        auto_archive_duration: threads.ThreadArchiveDuration,
+        auto_archive_duration: threads.ThreadArchiveDurationT,
```

Comment:
```suggestion
        auto_archive_duration: threads.ThreadArchiveDuration,
```
shouldn't change, it should just take an int/`ThreadArchiveDuration`